src/control/lib/spdk/ctests/nvme_discover_test.c

```diff
@@ -0,0 +1,174 @@
+/**
+* (C) Copyright 2018-2019 Intel Corporation.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*
+* GOVERNMENT LICENSE RIGHTS-OPEN SOURCE SOFTWARE
+* The Government's rights to use, modify, reproduce, release, perform, display,
+* or disclose this software are subject to the terms of the Apache License as
+* provided in Contract No. 8F-30005.
+* Any reproduction of computer software, computer software documentation, or
+* portions thereof marked with this legend must also reproduce the markings.
+*/
+
+#include <stdio.h>
+#include <stdarg.h>
+#include <stddef.h>
+#include <setjmp.h>
+#include <cmocka.h>
+
+#include "spdk/stdinc.h"
+#include "spdk/nvme.h"
+#include "spdk/env.h"
+
+#include "nvme_control.h"
+#include "nvme_control_common.h"
+
+/**
+ * ==============================
+ * nvme_control mock functions
+ * ==============================
+ */
+
+struct ret_t * __wrap_init_ret(void)
+{
+	return mock_ptr_type(struct ret_t *);
+}
+
+void __wrap_cleanup(void){
+	printf("Mock cleanup()...\n");
+}
+
+void __wrap_collect(struct ret_t *ret){
```

Comment:
(style)  open brace '{' following function declarations go on the next line
(style)  space required before the open brace '{'